apps/storybook/.storybook/docs-styling.css

```diff
@@ -1,33 +0,0 @@
-.sbdocs.sbdocs-h2 {
```

Comment:
op zich maakte deze custom styling de `.mdx` files wel wat leesbaarder

zou mss wel overwegen op een later moment, op de `details > summary` styling te behouden; standaard is het niet super duidelijk dat dit `expandable` is:

<img width="339" alt="image" src="https://github.com/milieuinfo/uigov-web-components/assets/15731348/7810cc18-6409-4832-a0ff-122854650030">

tov

<img width="346" alt="image" src="https://github.com/milieuinfo/uigov-web-components/assets/15731348/202f1411-3b08-4830-a221-80d8d89974c6">